Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/flow/HotDataFlow.kt

```diff
@@ -80,8 +81,11 @@ class HotDataFlow<T : Any>(
             }
         }
         .onCompletion { err ->
-            if (err != null) Timber.tag(tag).w(err, "internal onCompletion due to error")
-            else Timber.tag(tag).v("internal onCompletion")
+            when {
+                err is CancellationException -> Timber.tag(tag).d("internal onCompletion due to cancelation")
+                err != null -> Timber.tag(tag).w(err, "internal onCompletion due to error")
+                else -> Timber.tag(tag).v("internal onCompletion")
```

Comment:
Please include the `err.message` in the log? is not better to use `Timber.e` in this cases?

Reply:
The signature is `it: Throwable?`

* If it's a cancellation exception we don't care about the specific error message
* If it is an error we log it which would include it's error message automatically ( `.w(err,...)` ) 
* If it is neither of the above, then it was null and we have completion not caused by an exception.
* 
I'll adjust the log level to `e`